recipes/bootstrap.rb

```diff
@@ -1,6 +1,10 @@
 package 'git'
 package 'openssl'
 
+auth_data = data_bag_item node['vsts_agent']['data_bag'], node['vsts_agent']['data_bag_item']
```

Comment:
Could we make this recipe a little more flexible by allowing a node attribute for Test Kitchen, and then a `chef_vault_item` for production?

```suggestion
auth_data = node['vsts_agent']['pat'] ? node['vsts_agent']['pat'] : chef_vault_item node['vsts_agent']['data_bag'], node['vsts_agent']['data_bag_item']
```

We would need to depend on `chef-vault` in order to get the `chef_vault_item` helper, and update the `kitchen.yml` to interpolate environment variables to override the attribute. 

A benefit of the `chef_vault_item` helper is the fallback mechanism to `data_bag_item`.